tests/test_logging.py

```diff
@@ -38,6 +40,63 @@
 LOG_LEVELS = {"CRITICAL": 50, "ERROR": 40, "WARNING": 30, "INFO": 20, "DEBUG": 10}
 
 
+def test_add_instance():
+    # Testing adding an instance logger while checking if log has certain key
```

Comment:
Docstrings for defining the tests should go with ``"""content"""`` instead of ``# content``

Reply:
Thanks!